Contributing Page->Sir Lancebot: update description (#882)
discord-python/site: pydis_site/apps/content/resources/guides/pydis-guides/contributing/sir-lancebot.md

| `CHANNEL_ANNOUNCEMENTS` | ID of the `#announcements` channel |
| `CHANNEL_DEVLOG` | ID of the `#dev-log` channel |
| `CHANNEL_COMMUNITY_BOT_COMMANDS` | ID of the `#sir-lancebot-commands` channel |

> | `TRASHCAN_EMOJI` | The full emoji to use for the trashcan. Format should be like the output of `\:emoji:`. |

Reviewer: ```suggestion
| `TRASHCAN_EMOJI` | The full emoji to use for the trashcan, for example, `:wastebasket:` |
```
It might be worth mentioning a good default value, so people don't need to think about what to put.

Author: I think if the emoji is deafult on discord it needs its unicode and if it is a custom made emoji it needs its id. The output of sending `\:emoji:` just returns what is needed. 

So shall I still change it?

Reviewer: Ah, I didn't realise that, I don't think the current proposed text is that clear as it's not clear that it means the output when pasted into discord. We could just mention that they could set it to `🗑️` (use the unicode char directly).

It would be sort of nice to have something on sir-lancebot's side that made it use that by default in development, so users don't need to set it at all (fwiw the reason we have a custom on on the server is just that it looks nicer).

Author: Alright I will make the text more clear and make a pr on sir-lancebot repo for a default trash emoji

Author: @wookie184 is the new description good enough?

Reviewer: Yeah, looks good! We can probably remove it from the requirement vars now that the lancebot PR is merged though.
